docs/transparent-validation/sd-wire/faq.md

```diff
@@ -0,0 +1,35 @@
+# SDWire Frequently asked questions (FAQ)
+
+## Will the SDWire be compatible with the Odroid-XU4?
```

Comment:
@Stanislaw-bnk, this is incorrectly formulated. You use reverse logic. The question should be something like "How do you verify the compatibility of SDWire?" ort something like that.

Reply:
fixed in the latest commit